week3/homework/src/index.js

```diff
@@ -1,3 +1,84 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const Express = require('express');
+const {
+    createTodo,
+    deleteTodo,
+    readTheList,
+    updateTodo,
+    markAsDoneOrNotDone,
+    clearTodos
+} = require('./actions');
+
+const PORT = 3000;
+const app = Express();
+
+// request body parser
+app.use(Express.json());
+
+// createTodo 
+app.post('/todos', (req, res, next) => {
+    const { todo } = req.body;
+    createTodo(todo)
+        .then(data => res.send(data))
+        .catch(err => next(err))
+});
+
+// readList
+app.get('/todos', (req, res) => {
+    readTheList()
+        .then(data => res.send(data))
+        .catch(err => next(err));
```

Comment:
`next` is undefined.

Reply:
oh right, I added `next` as the third parameter.